src/Routes.jsx

```diff
@@ -1,40 +1,99 @@
 import React, { Component } from 'react';
-import {
-  graphql,
-  createFragmentContainer,
-} from 'react-relay';
+import DeckGL, { GeoJsonLayer, IconLayer } from 'deck.gl';
 import propTypes from 'prop-types';
+import { createFragmentContainer, graphql } from 'react-relay';
+import muniRoutesGeoJson from './res/muniRoutes.geo.json';
+
+const routesLayer = new GeoJsonLayer({
+  id: 'muni-routes-geojson',
+  data: muniRoutesGeoJson,
+  filled: true,
+  stroked: false,
+  extruded: true,
+});
 
 class Routes extends Component {
+  getMarkers() {
+    /* returns new DeckGL Icon Layer displaying all vehicles and stops on given routes */
+    const ICON_MAPPING = {
+      marker: {
+        x: 0, y: 0, width: 128, height: 128, mask: true,
+      },
+    };
+    const data = [];
+
+    // Push stop markers into data array
+    // because route.stops return null now, control stops before map function
+    this.props.state.routes.map(route => route.stops && route.stops.map(stop =>
```

Comment:
so if we're not putting the result of `map` into a list, we should be using `forEach` instead - see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach